articles/112_generated_interfaces_cpp.md

```diff
@@ -41,7 +41,7 @@ To separate the generated code from other code within the package it is defined
 Following the C++ style guide of ROS 2 the namespace hierarchy is mapped to a folder structure.
 The filenames use lowercase alphanumeric characters with underscores for separating words and end with either `.hpp` or `.cpp`.
 
-## Messages
+## <a name="Messages"></a>Messages
```

Comment:
This is not necessary. Each subsection already has an `id` and can be referenced, see http://design.ros2.org/articles/generated_interfaces_cpp.html#messages.

Reply:
I wasn't aware of that, thanks.  Fixed in 9b18608